LFS171x/fabric-material/tuna-app/README.md

```diff
@@ -1,9 +1,23 @@
-## Hyperledger Fabric Sample Application
+## Hyperledger Fabric Sample Application (Tuna App)
 
-This application demonstrates the creation and transfer of tuna fish shipments between actors leveraging Hyperledger Fabric in the supply chain. In this exercise we will set up the minimum number of nodes required to develop chaincode. It has a single peer and a single organization.
+    This application demonstrates the creation and transfer of tuna fish shipments between actors leveraging Hyperledger Fabric in the supply chain. In this exercise we will set up the minimum number of nodes required to develop chaincode. It has a single peer and a single organization.
```

Comment:
Don't want the initial indentation. This makes this not look right when viewing the README.